Sources/Bot/Services/MergeService.swift

```diff
@@ -677,7 +709,8 @@ extension MergeService.State: CustomDebugStringConvertible {
         let pullRequestsSeparator = "\n\t\t"
 
         let pullRequestsRepresentation = pullRequests.enumerated().map { index, pullRequest in
-            return "#\(index + 1): \(pullRequest)"
+            let isTP = pullRequest.isLabelled(withOneOf: self.topPriorityLabels)
+            return "#\(index + 1): \(pullRequest) \(isTP ? "[TP]" : "")"
```

Comment:
This is very debatable in my view. Do we really want to rely on hardcoded pseudo code strings?

Reply:
Not sure I'm following. This is only used to get a `debugDescription` of the queue of `MergeService` for the logs, what do you mean by "hardcoded _pseudo-code_ strings" 🤔?

Reply:
My mistake. I misread the code and thought they were part of the business logic. All good then.